tests/UI/package.json

```diff
@@ -9,6 +9,8 @@
     "// ESLint": "To run ESLint commands",
     "lint:fix": "eslint --fix --ignore-path .gitignore .",
     "lint": "eslint --ignore-path .gitignore .",
+    "// TypeScript Check": "Script to check typescript",
+    "check:typescript": "tsc --build ./tsconfig.json ",
```

Comment:
Add the `--noEmit` to avoid generating the build if we only want to check for errors ?

Reply:
Sorry but it's not possible 

`Compiler option '--noEmit' may not be used with '--build'.`